src/Microdown-DocumentBrowser/MicAbstractBlock.extension.st

```diff
@@ -9,7 +9,8 @@ MicAbstractBlock >> canSave [
 { #category : #'*Microdown-DocumentBrowser' }
 MicAbstractBlock >> contents [
 	"polymorphic with MicResourceReference - returns the source code of me - here generated"
-	^ MicTextualMicrodownExporter new visit: self
+	^ 'You cannot edit subsections of a document.
```

Comment:
I was thinking that raising an error would let the tool be able to propose something. 
Because here we would have to compare the contents. But I'm not sure.

Reply:
I think I made these changes based on an earlier discussion that the microdown parser should be "resilient" and not throw errors. 
I have gotten into some hard to identify bugs when the parser throws errors and give red-morph errors and broken inspectors. Of cause it is possible to catch, the debugging was just hard. 
I like this model better.

Reply:
OK I buy your take :)